bip-xyz.mediawiki

```diff
@@ -0,0 +1,141 @@
+    BIP: xyz
+    Layer: Consensus (soft fork)
+    Title: SIGHASH_NOINPUT
+    Author: Christian Decker <decker.christian@gmail.com
+    Comments-Summary: No comments yet.
+    Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-xyz
+    Status: Draft
+    Type: Standards Track
+    Created: 2017-02-28
+    License: PD
```

Comment:
PD is not an acceptable license for new BIPs. Please choose one from [this list](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#Recommended_licenses)